Classes/Command/StatusReportCommand.php

```diff
@@ -40,9 +40,7 @@ class StatusReportCommand extends Command
 
     public function __construct()
     {
-        parent::__construct();
-
-        $this->taskRepository = GeneralUtility::makeInstance(SchedulerTaskRepository::class);
+        parent::__construct('Start Report Command');
```

Comment:
This was not my idea. And why you have removed the taskRepository initialization?
Please change to:

```
public function __construct(?string $name = null)
{
    parent::__construct($name);

    $this->taskRepository = GeneralUtility::makeInstance(SchedulerTaskRepository::class);
}

Reply:
I just see you have implemented taskRepo with a setter. So, in that case the constructor can be removed completely

Reply:
okay